shared/src/prompts.rs

```diff
@@ -289,6 +289,7 @@ pub fn save_peer_invitation(
     keypair: KeyPair,
     server_api_addr: &SocketAddr,
     config_location: &Option<String>,
+    external_routing: bool,
```

Comment:
Clippy now complains about too many arguments here. This might be solved by constructing the `InterfaceConfig` before calling the function.

Reply:
That's ok, I can refactor that in another commit.